audit_history/utils.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from decimal import Decimal
+
+
+def json_formatter(o):
```

Comment:
A lot of these are very one-directional, though, correct? As in, a datetime is rendered into a string, but never turned back into a datetime?

Reply:
Yep. We put data and if this data can be serialized we return it if not we convert it to serializable data and return it